cs-lists/src/main/java/io/cloudslang/content/actions/ListContainsAction.java

```diff
@@ -0,0 +1,94 @@
+import static io.cloudslang.content.utils.Constants.OutputNames.RETURN_RESULT;
+import static io.cloudslang.content.utils.Constants.OutputNames.RETURN_CODE;
+import static io.cloudslang.content.utils.Constants.OutputNames.EXCEPTION;
+import static io.cloudslang.content.utils.Constants.OutputNames.RESPONSE;
+import static io.cloudslang.content.utils.Constants.ResponseNames.FAILURE;
+import static io.cloudslang.content.utils.Constants.ResponseNames.SUCCESS;
+import static io.cloudslang.content.utils.Constants.ReturnCodes.RETURN_CODE_FAILURE;
+import static io.cloudslang.content.utils.Constants.ReturnCodes.RETURN_CODE_SUCCESS;
+import static io.cloudslang.content.utils.Constants.TRUE;
+
+/**
+ * Created by moldovas on 7/12/2016.
+ */
+
+public class ListContainsAction {
+    private static final String SUBLIST = "sublist";
+    private static final String DELIMITER = "delimiter";
+    private static final String CONTAINER = "container";
+    private static final String IGNORE_CASE = "ignoreCase";
+
+    /**
+     * This method checks to see if a list contains every element in another list.
+     *
+     * @param sublist      The contained list.
+     * @param container    The containing list.
+     * @param delimiter    A delimiter separating elements in the two lists. Default is a comma.
+     * @param ignoreCase   If set to 'True' then the compare is not case sensitive. Default is True.
+     * @return sublist is contained in container or not.
+     */
+
+    @Action(name = "List Contains All",
+            outputs = {
+                    @Output(RESPONSE_TEXT),
+                    @Output(RETURN_RESULT),
+                    @Output(RETURN_CODE),
+                    @Output(EXCEPTION)
+            },
+            responses = {
+                    @Response(text = SUCCESS, field = RETURN_CODE, value = RETURN_CODE_SUCCESS, matchType = MatchType.COMPARE_EQUAL),
+                    @Response(text = FAILURE, field = RETURN_CODE, value = RETURN_CODE_FAILURE, matchType = MatchType.COMPARE_EQUAL, isOnFail = true, isDefault = true)
+            })
+    public Map<String, String> containsElement(@Param(value = SUBLIST, required = true) String sublist,
+                                               @Param(value = CONTAINER, required = true) String container,
+                                               @Param(value = DELIMITER) String delimiter,
+                                               @Param(value = IGNORE_CASE) String ignoreCase
+                                               ) {
+        Map<String, String> result = new HashMap<>();
+        ignoreCase = getInputDefaultValue(ignoreCase, "true");
+        try {
+            String[] subArray = sublist.split(delimiter);
+            String[] containerArray = container.split(delimiter);
+            String[] uncontainedArray = ListProcessor.getUncontainedArray(subArray, containerArray, Boolean.parseBoolean(ignoreCase));
```

Comment:
If the user provides a string different than true/false/TRUE/FALSE, the ignoreCase value will be false, although you want  it to be true, because this is the default. I think you could do one of the following:
1) Do some additional logic for parsing a boolean with the default value true
2) Specify the ignoreCase argument type as boolean insted of String. This way, CS platform will validate that the input is indeed a boolean and will throw a value otherwise